python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pytorch_lightning import LightningModule
+import onnxruntime as ort
+from functools import partial
+import warnings
+import torch
+import math
+import numpy as np
+
+
+ONNXRT_BINDED_COMPONENTS = ['_ortsess_up_to_date',
+                            '_ortsess',
+                            '_build_ortsess',
+                            'update_ortsess',
+                            'predict_step',
+                            'inference']
+
+
+def bind_onnxrt_methods(pl_model: LightningModule):
+    # class type check
+    assert isinstance(pl_model, LightningModule),\
+        "onnxruntime support is only valid for a LightningModule."
+
+    # check conflicts
+    for component in ONNXRT_BINDED_COMPONENTS:
+        if component in dir(pl_model):
+            warnings.warn(f"{component} method/property will be replaced.")
```

Comment:
Maybe explaining why this happened and what users can do to make this warning disappear. If there is nothing user can do, maybe it is better to use `logging.warn`.

![image](https://user-images.githubusercontent.com/10561966/142821073-070e89f9-d231-477a-ab63-00eb4326c8c6.png)


see https://docs.python.org/3/howto/logging.html